### DISCUSSION TITLE
AOSP 12.0 v400.e

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2119] [General]
- [It's Christmas](https://www.youtube.com/watch?v=GsRJ-iRZasQ) (the link is totally unrelated with anything).
- Christmas animation, thanks @sepehrmoh81
- UDFPS working for various devices. I counted OnePlus Nord 2 and Samsung Galaxy A51, probably others
- Uh, various stuff I'm too lazy to rememeber

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v400.e'>AOSP 12.0 v400.e</a>.</em>

**Comment by rvlphee** [comment]
Why it's not Rick roll link?

**Reply by aryan0078** [reply]
Volte bug in M31 samsung exynos

**Reply by tosuman** [reply]
same problem sir

**Comment by tosuman** [comment]
I've the issue with VOLTE network Moto one action

**Comment by mmbb12** [comment]
👍

**Comment by RFX-LEGEND** [comment]
### Moto X4
#### Experience
Pretty easy installation and not many problems other than some lag which is expected from pre-releases.
#### Bugs
- not detecting sim

**Comment by ghost** [comment]
Rip for A only :(

**Comment by devrieshh** [comment]
the square menu sign still doesnt work(triangle - point - square)on the Cubot king kong mini2     system-squeak-arm32_binder64-ab-vndklite-gogapps-secure.img.xz

![IMG_20211229_184630](https://user-images.githubusercontent.com/7243128/147689991-f309452c-f447-4c55-b923-3f923e6b2963.jpg)

**Reply by devrieshh** [reply]
in  system-squeak-arm32_binder64-ab_vanilla AOSP 12.0 v400.g  the whole menu is working

**Reply by devrieshh** [reply]
in the system-squeak-arm32_binder64-ab-vndklite-gogapps-secure.img still not

**Comment by phhusson** [comment]
try system-squeak-arm32_binder64-ab_vanilla

**Comment by devrieshh** [comment]
i will try later because everything else is working fine now

**Comment by murdika** [comment]
Nice rom,boot on my device asus zenfone live l1,but have issue with mobile data internet, i hope next build will fix,
I use vndklite with pie-lite vendor
Thanks for the awesome rom.

**Comment by Reddy2903** [comment]
UDFPS doesn't work on my Nord CE. And VoLTE has to be reset everytime the device reboots/signal type changes from 3G/2G to LTE.
Is there an option to enable UDFPS specifically??
